src/components/default-code-snippet/default-code-snippet.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import * as React from 'react';
+import { MuiThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+import { CodeSnippet, CodeSnippetDataProps } from '~/components/code-snippet/code-snippet';
+import grey from '@material-ui/core/colors/grey';
+
+const theme = createMuiTheme({
+    overrides: {
```

Comment:
It may be possible to do 
```
pallette: {background: {default: color}}
``` 
instead of overrides.

Reply:
I prefer overrides because I change only one material component.